airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -63,6 +63,14 @@
   dockerImageTag: 0.1.20
   documentationUrl: https://docs.airbyte.io/integrations/destinations/gcs
   icon: googlecloudstorage.svg
+- name: Google Firestore
+  destinationDefinitionId: 27dc7500-6d1b-40b1-8b07-e2f2aea3c9f4
+  dockerRepository: airbyte/destination-firestore
+  dockerImageTag: 0.1.1
+  # todo - https://github.com/airbytehq/airbyte/issues/9727
+  documentationUrl: https://docs.airbyte.io/integrations/destinations/firestore
+  icon: firestore.svg
+  sourceType: api
```

Comment:
```suggestion
```
I don't think this is a valid field for a destination definition :)

Reply:
Yeah I figured out late 🙄